tests/rust/raw_lines.toml

```diff
@@ -0,0 +1,3 @@
+include_guard = "INCLUDE_GUARD_H"
+no_includes = true
```

Comment:
Let's test that this works actually after the includes?

Reply:
Sure, patch updated